src/components/layout/DevconGrantsLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import { Stack } from '@chakra-ui/react';
+import { FC } from 'react';
+
+import { AcademicGrantsHero } from '../UI';
+
+export const DevconGrantsLayout: FC = ({ children }) => {
+  return (
+    <Stack>
+      <AcademicGrantsHero title='Devcon Grants Round'>
+        In an effort to promote edcuation around the Ethereum in Colombia and Latin America in
```

Comment:
typo here, 'edcuation'. Shouldn't this copy be "In an effort to support the growth of the Ethereum community..." like the old page copy?

Reply:
Good catch, fixing the typo. This is an updated text shared by Skylar.